### PR TITLE
[Backport][Android] Use JNI instead NDK MediaCrypto for secure decoder check

### PR DESCRIPTION
**Pull request by peak3d**
## Description
Backport of #16729

### CODE REVIEW

File: tools/depends/target/libandroidjni/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=libandroidjni
-VERSION=aa12538cc5090d061d34b5fd7385d939ea82ce8b
+VERSION=fc24c9cc507a244a587fa590f6882a588b9fc927
```
Comment (Rechi) [inline tools/depends/target/libandroidjni/Makefile]: Does this PR rely need the following changes https://github.com/xbmc/libandroidjni/compare/aa12538...25258a1?

Comment (peak3d) [reply tools/depends/target/libandroidjni/Makefile]: This PR needs the changes of fc24c9cc507a244a587fa590f6882a588b9fc927

Comment (Rechi) [reply tools/depends/target/libandroidjni/Makefile]: Yes this PR needs the code changes of commit https://github.com/xbmc/libandroidjni/commit/fc24c9c. But does it need the code changes happening in https://github.com/xbmc/libandroidjni/compare/aa12538...25258a1 (memory and hdr info)?

Comment (peak3d) [reply tools/depends/target/libandroidjni/Makefile]: No, it doesn't need them, now I'm curious about your suggestion....

Comment (Rechi) [reply tools/depends/target/libandroidjni/Makefile]: Then use a version which is based on https://github.com/xbmc/libandroidjni/commit/aa12538 and includes the code changes of https://github.com/xbmc/libandroidjni/commit/fc24c9c.

**Comment by Rechi** [comment]
@peak3d please always set the version label before or right after merging a PR.

**Pull request merged**